rascal-vscode-extension/src/fs/RascalFileSystemProviders.ts

```diff
@@ -47,6 +47,27 @@ export class RascalFileSystemProvider implements vscode.FileSystemProvider {
         });
     }
 
+    sendRequest<R>(uri : vscode.Uri, method: string): Promise<R>;
+    sendRequest<R, A>(uri : vscode.Uri, method: string, param: A): Promise<R>;
+    sendRequest<R, A>(uri : vscode.Uri, method: string, param?: A): Promise<R> {
+        return this.client.sendRequest<R>(method, param ?? { uri: uri.toString()} )
+            .catch((r: ResponseError) => {
+                if (r !== undefined) {
+                    this.client.info("Got responds error from the file system: ", r);
```

Comment:
```suggestion
                    this.client.info("Got response error from the file system: ", r);
```

Reply:
Thanks, fixed. And also made this a `debug` msg instead of `info`.